internal/bundle.go

```diff
@@ -110,5 +108,15 @@ func (bundle *Bundle) FinishComposing() (TarFileSets, error) {
 }
 
 func (bundle *Bundle) getFileRelPath(fileAbsPath string) string {
-	return utility.PathSeparator + utility.GetSubdirectoryRelativePath(fileAbsPath, bundle.Directory)
+	return utility.GetSubdirectoryRelativePath(fileAbsPath, bundle.Directory)
```

Comment:
Why to remove the leading path separator?

Reply:
Make behaviour like in postgres relative path